src/Slider/Slider.jsx

```diff
@@ -23,6 +23,7 @@ const Slider = class Slider extends React.Component {
     isPlaying: PropTypes.bool.isRequired,
     lockOnWindowScroll: PropTypes.bool.isRequired,
     masterSpinnerFinished: PropTypes.bool.isRequired,
+    moveThreshold: PropTypes.number.isRequired,
```

Comment:
If we are going to have a defaultProp for it and it says in the table it is not required, shouldn't this not have `isRequired`?

Reply:
I removed `isRequired`.